src/routes/misc.ts

```diff
@@ -10,8 +10,6 @@ router.get('/transformerBuildVersion', MiscController.buildVersion); // depricia
 router.get('/buildVersion', MiscController.buildVersion);
 router.get('/version', MiscController.version);
 router.get('/features', MiscController.features);
```

Comment:
what will be impact or changes to do for existing use-cases/dashboards depending on these routes ?